src/python/pants/engine/internals/examples/parsers.py

```diff
@@ -202,96 +200,3 @@ def encode_json(obj, inline=False, **kwargs):
     """
     encoder = JSONEncoder(default=functools.partial(_object_encoder, inline=inline), **kwargs)
     return encoder.encode(obj)
-
-
-class PythonAssignmentsParser(Parser):
-    """A parser that parses the given python code into a list of top-level objects found.
-
-    Only Serializable objects assigned to top-level variables will be collected and returned.  These
-    objects will be addressable via their top-level variable names in the parsed namespace.
```

Comment:
As conceptually clean as this was, it couldn't possibly work with the new macros. For the best I think.